client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/allocation/useAllocationContext.ts

```diff
@@ -112,15 +116,10 @@ export const useAllocationContext = create<AllocationContext>((set, get) => ({
       isDirty: false,
       item,
 
-      allocateIn:
-        item.isVaccine && allocateVaccineItemsInDoses
-          ? AllocateIn.Doses
-          : AllocateIn.Units,
```

Comment:
Won't initialise in doses (post convo with dhanya - if you're doing shipments, you're probably thinking in units/packs) but doses option will be available in the dropdown